webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/ScrambleRequest.java

```diff
@@ -533,7 +535,10 @@ private static void addScrambles(PdfWriter docWriter, Document doc, ScrambleRequ
                 ct.go();
 
                 doc.newPage();
+
+                addScrambleSheet(doc, scrambleRequest, globalTitle, scramble, cb, i);
             }
+            addSolutionSheet(docWriter, doc, scrambleRequest, globalTitle);
```

Comment:
This is the generic FMC solution sheet that does not have a scramble on it, right? Is it possible that we end up calling this method twice if we generate 2 rounds of FMC?